src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class TaskList {
+    final static BadCommandException INDEX_OOB_ERROR =
+            new BadCommandException("Index given is out of bounds!");
```

Comment:
Maybe this can be declared within badcommandexception class and be used as a static exception (i.e. Using `BadCommandException.INDEX_OOB_ERROR`)